service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java

```diff
@@ -108,6 +113,155 @@ public void recordStewardship(
+    siteEntriesRecord.setSiteNotes(addSiteRequest.getSiteNotes());
+
+    siteEntriesRecord.store();
+  }
+
+  private List<SiteEntry> getSiteEntries(int siteId) {
+    List<SiteEntriesRecord> records =
+        db.selectFrom(SITE_ENTRIES)
+            .where(SITE_ENTRIES.SITE_ID.eq(siteId))
+            .orderBy(SITE_ENTRIES.UPDATED_AT)
+            .fetch();
+
+    List<SiteEntry> siteEntries = new ArrayList<>();
+
+    records.forEach(
+        record -> {
+          String username;
+          if (record.getUserId() == null) {
+            username =
+                db.selectFrom(ENTRY_USERNAMES)
+                    .where(ENTRY_USERNAMES.ENTRY_ID.eq(record.getId()))
+                    .fetchOne(ENTRY_USERNAMES.USERNAME);
+          } else {
+            username =
+                db.selectFrom(USERS)
+                    .where(USERS.ID.eq(record.getUserId()))
+                    .fetchOne(USERS.USERNAME);
+          }
+
+          SiteEntry siteEntry =
+              new SiteEntry(
+                  record.getId(),
+                  username,
+                  record.getUpdatedAt(),
+                  record.getTreePresent(),
+                  record.getStatus(),
+                  record.getGenus(),
+                  record.getSpecies(),
+                  record.getCommonName(),
+                  record.getConfidence(),
+                  record.getDiameter(),
+                  record.getCircumference(),
+                  record.getMultistem(),
+                  record.getCoverage(),
+                  record.getPruning(),
+                  record.getCondition(),
+                  record.getDiscoloring(),
+                  record.getLeaning(),
+                  record.getConstrictingGate(),
+                  record.getWounds(),
+                  record.getPooling(),
+                  record.getStakesWithWires(),
+                  record.getStakesWithoutWires(),
+                  record.getLight(),
+                  record.getBicycle(),
+                  record.getBagEmpty(),
+                  record.getBagFilled(),
+                  record.getTape(),
+                  record.getSuckerGrowth(),
+                  record.getSiteType(),
+                  record.getSidewalkWidth(),
+                  record.getSiteWidth(),
+                  record.getSiteLength(),
+                  record.getMaterial(),
+                  record.getRaisedBed(),
+                  record.getFence(),
+                  record.getTrash(),
+                  record.getWires(),
+                  record.getGrate(),
+                  record.getStump(),
+                  record.getTreeNotes(),
+                  record.getSiteNotes());
+
+          siteEntries.add(siteEntry);
+        });
+
+    return siteEntries;
+  }
+
+  @Override
+  public GetSiteResponse getSite(int siteId) {
+    SitesRecord sitesRecord = db.selectFrom(SITES).where(SITES.ID.eq(siteId)).fetchOne();
```

Comment:
You should null check this record incase the siteId doesn't exist

Reply:
+1

Reply:
Fixed